common/static/js/src/accessibility_tools.js

```diff
@@ -33,7 +33,111 @@ NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVE
 ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 */
-var $focusedElementBeforeModal;
+
+var $focusedElementBeforeModal,
+    focusableElementsString = 'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, *[tabindex], *[contenteditable]';
+
+var reassignTabIndexesAndAriaHidden = function(focusableElementsFilterString, closeButtonId, modalId, mainPageId) {
+    // Sets appropriate elements to tab indexable and properly sets aria_hidden on content outside of modal
+    // "focusableElementsString" is a string that indicates all elements that should be focusable
```

Comment:
this should be `focusableElementsFilterString` to match the function contract above